docs/src/main/docs/guides/07_health_se_guide.adoc

```diff
@@ -0,0 +1,624 @@
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Health Check SE Guide
+:description: Helidon health-checks
+:keywords: helidon, health-check, health, check
+
+This guide describes how to create a sample Helidon SE project
+that can be used to run some basic examples using both built-in and custom health-checks.
+
+== What you need
+
+[width=50%,role="flex, sm7"]
+|===
+|About 15 minutes
+|<<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|===
+
+=== Create a sample SE project
+
+Generate the project sources using the Helidon SE Maven archetype.
+The result is a simple project that can be used for the examples in this guide.
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype:
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-se \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-se \
+    -Dpackage=io.helidon.examples.quickstart.se
+----
+
+=== Using the built-in health-checks
+
+Helidon has a set of built-in health-checks that can be optionally enabled to report various
+ health-check statuses that are commonly used:
+
+* deadlock detection
+* available disk space
+* available heap memory
+
+The following example will demonstrate how to use the built-in health-checks.  These examples are all executed
+from the root directory of your project (helidon-quickstart-se).
+
+[source,xml]
+.Notice that the built-in health-check dependency is already in the project's pom.xml file:
+----
+<dependency>
+    <groupId>io.helidon.health</groupId>
+    <artifactId>helidon-health-checks</artifactId>
+</dependency>
+----
+
+[source,java]
+.Modify `Main.java`, import `HealthCheckReponse`, and replace the `createRouting` method:
+----
+
+import org.eclipse.microprofile.health.HealthCheckResponse;  // <1>
+
+...
+
+private static Routing createRouting(Config config) {
+
+    HealthSupport health = HealthSupport.builder()
+      .addLiveness(HealthChecks.healthChecks())  // <2>
+      .build();
+
+    return Routing.builder()
+      .register(JsonSupport.create()) // <3>
+      .register(health)  // <4>
+      .build();
+}
+----
+<1> Import `HealthCheckResponse`.
+<2> Add built-in health-checks (requires the `helidon-health-checks`
+ dependency).
+<3> Register the `JSON-P` support in the WebServer routing.
+<4> Register the created health support with web server routing (adds the
+`/health` endpoint).
+
+
+[source,bash]
+.Build the application, skipping unit tests, then run it:
+----
+mvn package -DskipTests=true
+java -jar target/helidon-quickstart-se.jar
+----
+
+[source,bash]
+.Verify the health endpoint in a new terminal window:
+----
+curl http://localhost:8080/health
+----
+
+[source,json]
+.JSON response:
+----
+{
+    "outcome": "UP",
```

Comment:
It looks like this was run with an older version of Helidon SE? When I run using the 1.2.1 archetype I get both `outcome` and `status` (and `state` and `status`) just like with MP Health. 

So a similar NOTE applies for SE as well ("you should rely on status")

Reply:
fixed